Sources/FileSystem/Model/BetaGroup.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+public struct BetaGroup: Codable, Equatable {
+
+    public typealias EmailAddress = String
+
+    public var id: String?
+    public var groupName: String
+    public var isInternal: Bool?
+    public var publicLink: String?
+    public var publicLinkEnabled: Bool?
+    public var publicLinkLimit: Int?
+    public var publicLinkLimitEnabled: Bool?
+    public var creationDate: String?
+    public var testers: [EmailAddress]
```

Comment:
🤔 I'm not sure any of these properties need to be mutable when we read them from or store them to disk